src/framework-ondata.js

```diff
@@ -52,14 +55,14 @@ module.exports = function(payload, socketIndex) {
              throw Error( 'No wrapper fRunUpdater function found!  Cannot run updater.' );
 
          } else { 
-             // TODO: revisit this logic later, per rdf-pipeline/noflo-rdf-pipeline#35
 
+             // TODO: revisit this logic later, per rdf-pipeline/noflo-rdf-pipeline#35
              // save data used to see if we have a different state after running updater
- 
              clearStateData( vni.errorState ); // clear last error state
 
              // Save wrapper to make it accessible from the Promise
              var wrapper = this.nodeInstance.wrapper;
+             var updaterStartTime = Date.now();
```

Comment:
I think it would be good to call a profiler function here, to track the difference between the number of entries vs exits.